frontend/src/app/libs/landing/components/events-page/events-page.component.html

```diff
@@ -1,12 +1,26 @@
-<ng-container *ngIf="events.status === successStatus">
-	<app-user-events-list
-		*ngIf="user?.value"
-		[user]="user.value"
-		[events]="events.value">
-	</app-user-events-list>
+<div class="app-landing-page">
```

Comment:
i think this "clouds" panel must be visible only for not registered users.
If user is done sign in this block must be removed from page, so please use ngIf here